src/Maths/Silk.NET.Maths/PublicAPI/net5.0/PublicAPI.Shipped.txt

```diff
@@ -295,7 +295,6 @@ Silk.NET.Maths.Matrix3X3<T>.M33.set -> void
 Silk.NET.Maths.Matrix3X3<T>.Matrix3X3() -> void
 Silk.NET.Maths.Matrix3X3<T>.Matrix3X3(Silk.NET.Maths.Matrix2X4<T> value) -> void
 Silk.NET.Maths.Matrix3X3<T>.Matrix3X3(Silk.NET.Maths.Matrix3X2<T> value) -> void
-Silk.NET.Maths.Matrix3X3<T>.Matrix3X3(Silk.NET.Maths.Matrix3X3<T> value) -> void
```

Comment:
Sorry, drawing the line at removing APIs from a shipping release. Everything else is fine though.